src/alto-ui/List/List.scss

```
@@ -1,62 +1,98 @@
-@import "../scss/inc";
+@import '../scss/inc';
 
-.list {
+.List {
   background: $white;
   list-style: none;
   overflow: auto;
 }
 
-.list__item {
-  border: 1px solid $coolgrey-20;
-  border-top: 0;
-  overflow: hidden;
+.List__item {
+  padding: 0;
+  position: relative;
 
-  &:first-child {
-    border-top: 1px solid $coolgrey-20;
-    border-top-left-radius: $border-radius-default;
-    border-top-right-radius: $border-radius-default;
-  }
-
-  &:last-child {
-    border-bottom-left-radius: $border-radius-default;
-    border-bottom-right-radius: $border-radius-default;
+  &--active {
+    background-color: $blue-10;
   }
 }
 
-
-.list__item-content {
-  padding: .75rem 1.25rem;
+.List__item-row {
   display: flex;
-  flex-direction: row;
-  text-decoration: none;
+  padding: 0 $spacing-small;
   align-items: center;
-  font-weight: $font-weight-semibold;
-  color: $color-text-default;
-  justify-content: space-between;
+  min-height: 4rem;
+
+  &--small {
+    min-height: 2rem;
+    padding: 0 0.32rem;
+  }
+}
+
+.List__item-nested-row {
+  margin-left: $spacing-large;
```

When using ``borderless`` in NestedList, to have the same spacing on the right

```suggestion
  margin-left: $spacing-large;
  margin-right: $spacing-large;
```